docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -110,7 +110,7 @@ is used.
   * - ``iceberg.max-partitions-per-writer``
     - Maximum number of partitions handled per writer.
     - 100
-  * - ``hive.orc.bloom-filters.enabled``
+  * - ``iceberg.orc.bloom-filters.enabled``
```

Comment:
Can we make this one as the first commit ? So that we don't have to do this rename.

Reply:
Done.